app/experimenter/nimbus-ui/src/hooks/useOutcomes.test.tsx

```diff
@@ -56,6 +58,23 @@ describe("hooks/useOutcomes", () => {
     expect(result.current.primaryOutcomes).toStrictEqual([]);
     expect(result.current.secondaryOutcomes).toStrictEqual([]);
   });
+
+  it("returns an array of outcomes that are eligible for that experiment based on application", () => {
+    const { mock, experiment } = mockExperimentQuery("howdy", {
+      application: NimbusExperimentApplication.FENIX,
+    });
+    const { result } = renderHook(() => useOutcomes(experiment), {
+      wrapper,
+      initialProps: { mocks: [mock] },
+    });
+
+    // Outcomes 1-3 have DESKTOP applications
```

Comment:
Might be worth explicitly filtering these for desktop vs fenix. But that's probably just duplicating the tested functionality, so 🤷

Reply:
That's a good call, mocks could change